packages/graph-explorer-proxy-server/node-server.js

```diff
@@ -32,59 +32,70 @@ const getCredentials = async () => {
 };
 
 dotenv.config({ path: "../graph-explorer/.env" });
-  
+
 (async () => {
   let creds = await getCredentials();
   let requestSig;
 
   app.use(cors());
 
-  app.use("/explorer", express.static(path.join(__dirname, "../graph-explorer/dist")));
+  app.use(
+    "/explorer",
+    express.static(path.join(__dirname, "../graph-explorer/dist"))
+  );
 
-  const delay = (ms) => new Promise((resolve) => setTimeout(() => resolve(), ms));
+  const delay = (ms) =>
+    new Promise((resolve) => setTimeout(() => resolve(), ms));
 
-  async function retryFetch (
+  async function retryFetch(
     url,
     retries = 1,
     retryDelay = 10000,
     req,
     language
   ) {
-
     let reqObjects;
 
     if (creds === undefined) {
       console.error("Credentials undefined. Trying refresh.");
       creds = await getCredentials();
       if (creds === undefined) {
-        throw new Error("Credentials undefined after refresh. Check that you have proper access and that the credentials should work.")
+        throw new Error(
+          "Credentials undefined after refresh. Check that you have proper access and that the credentials should work."
+        );
       }
     }
-    reqObjects = await getRequestObjects(req.headers["graph-db-connection-url"], req.headers["aws-neptune-region"]);
+    reqObjects = await getRequestObjects(
+      req.headers["graph-db-connection-url"],
+      req.headers["aws-neptune-region"]
+    );
     await getAuthHeaders(language, req, reqObjects[0], reqObjects[2]);
 
     return new Promise((resolve, reject) => {
-        
       const wrapper = (n) => {
-        fetch(url, { headers: req.headers, })
-          .then(async res => {
-
+        fetch(url, { headers: req.headers })
+          .then(async (res) => {
             if (!res.ok) {
-	            console.log("Response not ok");
-              const error = res.status
+              console.log("Response not ok");
+              const error = res.status;
               return Promise.reject(error);
             } else {
-	            console.log("Response ok");
+              console.log("Response ok");
```

Comment:
I recommend having separate PR/commits for styling changes and business logic changes in the future since the one for styling/formatting changes can be quickly reviewed allowing reviewer to focus on business logic changes only.

Reply:
It was the pre-commit process configuration. It ran lint and prettier automatically on every modified file. It prevents wrong styling issues, so I need to figure out why in the previous update it didn't update the styling.

However, I could remove all those styling changes manually and prevent the pre-commit hook for this time.

Reply:
@nestoralvarezd can you please make an update to remove styling changes in this PR? Once approved, we can add the styling changes.